sdk/core/azure-core-experimental/src/main/java/com/azure/core/experimental/geojson/GeoBoundingBox.java

```diff
@@ -1,14 +1,14 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
-package com.azure.core.experimental.spatial;
+package com.azure.core.experimental.geojson;
 
 import java.util.Objects;
 
 /**
  * Represents a geometric bounding box.
  */
-public final class GeometryBoundingBox {
+public final class GeoBoundingBox {
```

Comment:
For the bounding box the RFC indicates that it must contain 2*N coordinates where N is equivalent to the number of coordinates in the geometry it is bounding. A 2D geometry requires the bounding box to have 4 coordinates, 3D requires 6.

https://tools.ietf.org/html/rfc7946#section-5

@pakrym, @KrzysztofCwalina, @JonathanGiles, @srnagar, would this be something we want to validate? I know our guidelines generally state to limit client side validation, is this more in the context of a service requires or does it apply more broadly to RFC guidelines/requirements as well?

Reply:
I think we should go with our guidance of not validating too much. We can let cosmos service decide whether it wants to accept these geometries.

Reply:
I agree. Let's not do too much validation here. Service can do this validation.